app/views/reviews/new.html.erb

```diff
@@ -7,7 +7,7 @@
   <%= text_field_tag :title %><br>
 
   <%= label_tag :rating %>
-  <%= select_tag :rating, options_for_select([1, 2, 3, 4, 5]) %><br>
+  <%= text_field_tag :rating %><br>
```

Comment:
changed this because when we got to the edit form, it need to be pre-filled with the already given entries. Couldn't figure out how to pre-fill select options. Feel free to switch that out if you can.

Reply:
Not a problem, I like this as well. We will might to consider putting in some precautions for edge cases, such as the user entering a character instead of an integer.